dbt/include/databricks/macros/adapters/python.sql

```diff
@@ -72,7 +72,7 @@ writer.saveAsTable("{{ target_relation }}")
     {%- endif %}
 .partitionBy({{ partition_by }})
 {%- endif -%}
-{%- if liquid_clustered_by -%}
+{%- if liquid_clustered_by and not is_incremental() -%}
```

Comment:
if is_incremental, it is pointless to liquid cluster by, as this is a temp table that will be merged into the table that we actual want to liquid cluster by, and optimize will not be called before then.